packages/plans-grid/src/plans-interval-toggle/index.tsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { useI18n } from '@automattic/react-i18n';
+import { sprintf } from '@wordpress/i18n';
+import { Popover } from '@wordpress/components';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import SegmentedControl from '../segmented-control';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+// TODO: import this type directly from plans data-store once
+// https://github.com/Automattic/wp-calypso/pull/48790 is merged
+export type BillingIntervalType = 'MONTHLY' | 'ANNUALLY';
+
+export const PopupMessages: React.FunctionComponent = ( { children } ) => {
+	const variants: Record< string, React.ComponentProps< typeof Popover >[ 'position' ] > = {
+		desktop: 'middle right',
+		mobile: 'bottom center',
+	};
+
+	return (
+		<>
+			{ Object.keys( variants ).map( ( variant ) => (
+				<Popover
+					key={ variant }
+					className={ classNames(
+						'plans-interval-toggle__popover',
+						`plans-interval-toggle__popover--${ variant }`
+					) }
+					position={ variants[ variant ] }
+					noArrow={ false }
+				>
+					{ children }
+				</Popover>
+			) ) }
+		</>
+	);
+};
+
+type ToggleHostProps = {
+	intervalType: BillingIntervalType;
+	onChange: ( selectedValue: BillingIntervalType ) => void;
+	maxSavingsPerc: number;
+	className?: string;
+};
+
+const PlansIntervalToggle: React.FunctionComponent< ToggleHostProps > = ( {
+	onChange,
+	intervalType,
+	maxSavingsPerc,
+	className = '',
+} ) => {
+	const { __ } = useI18n();
+
+	return (
+		<div
+			className={ classNames(
+				'plans-interval-toggle',
+				{ 'plans-interval-toggle--monthly': intervalType === 'MONTHLY' },
+				className
+			) }
+		>
+			<SegmentedControl>
+				<SegmentedControl.Item
+					selected={ intervalType === 'MONTHLY' }
+					onClick={ () => onChange( 'MONTHLY' ) }
+				>
+					<span className="plans-interval-toggle__label">
+						{ __( 'Monthly', __i18n_text_domain__ ) }
+					</span>
+				</SegmentedControl.Item>
+
+				<SegmentedControl.Item
+					selected={ intervalType === 'ANNUALLY' }
+					onClick={ () => onChange( 'ANNUALLY' ) }
+				>
+					<span className="plans-interval-toggle__label">
+						{ __( 'Annually', __i18n_text_domain__ ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Pay annually' )` *ES Score: 8*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/feat/plans-grid-billing-period-toggle)